webpack.config.js

```diff
@@ -0,0 +1,44 @@
+const path = require('path');
```

Comment:
I see that in TS file we have 2 spaces indentation. Why so we have 4 spaces indentation in JS files?

Reply:
Fixed